Dockerfile

```diff
@@ -40,7 +40,8 @@ RUN set -x && \
         # workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1398272
         rpm -q dnf && \
         # some unknown thing
-        dnf -y mark install $(dnf repoquery --unneeded -q); \
+        UNNEEDED=$(dnf repoquery --unneeded -q | xargs) && \
```

Comment:
`dnf -q repoquery --unneeded | xargs --no-run-if-empty dnf mark install`

Reply:
Thanks, this looks better.